javascript/bunicorn/config.yaml

```diff
@@ -1,6 +1,6 @@
 framework:
   github: ragokan/bunicorn
-  version: 0.1
+  version: 0.7
```

Comment:
```suggestion
  version: 0.0
```

I didn't notice last time

Reply:
Oh, so I will do this on next pull request since you merged. Thanks!

Reply:
I thin I'll make by han'd for next update